### DISCUSSION TITLE
Compilation failure in Windows

### DISCUSSION THREAD

**Discussion by SamaHans** [OpenRadioss/OpenRadioss#903] [Q&A] [answered]
Does anyone encounter this problem while compiling in Windows?

<img width="945" alt="捕获" src="https://user-images.githubusercontent.com/12651419/207608915-d44e0e2c-ac65-4360-b6d6-13a60cdea937.PNG">

**Comment by MarianBulla** [comment, marked as answer]
Hello, 
can you check, if you 
- use the recomended VisualStudio libs
- Intel oneApi 
- cygwin (without link.exe)
please ?
On my side it works fine (see picture).
I'm using windows 10 (64bit) + VisualStudi2019 + Intel oneAPI2022.
Thanks and best,
Marian
![image](https://user-images.githubusercontent.com/105412502/207851678-408bb92f-3178-47b3-bc26-0063d7be7e89.png)

**Reply by SamaHans** [reply]
Thanks for your kind reply! However, I realized that the Fortran and C++ compiler are automatically identified as GNU, how to fix it?

**Reply by MarianBulla** [reply]
This is, how I start my cygwin session (see image below).

![image](https://user-images.githubusercontent.com/105412502/207892601-da7f88cf-00dd-487f-b67a-53e622826476.png)
![image](https://user-images.githubusercontent.com/105412502/207892701-24653a7c-a893-48f1-8180-a140cd696224.png)

**Reply by SamaHans** [reply]
By add "-DCMAKE_C_COMPILER=icx" and "-DCMAKE_CXX_COMPILER" in CMakeLists.txt, the problem is fixed. Thank you so much!

**Reply by MarianBulla** [reply]
This is great !
Thank you very much for all your efforts ! 
Always great to have such smart people, like you, on board :-)
Please let me know about your further progress with the models and don't hesitate to communicate broadly, within this board.
Thanks and best regards
Marian

**Reply by SamaHans** [reply]
Thanks, Marian! However, when compiling engine, error comes as
![image](https://user-images.githubusercontent.com/12651419/208055338-ae65d089-5a85-4035-8508-5a3876d25d9e.png)
I have tried modify the Cmake files, like setting CMAKE_CXX_FLAGS=-fexceptions, but it's not working. It would be great if you can share with this.

**Reply by SamaHans** [reply]
Fine, done! I have reached the conclusion that one must compile with oneAPI later than 2022.1, otherwise various problems can show up.
